central/usage/csv/csv.go

```diff
@@ -0,0 +1,39 @@
+package usagecsv
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+var (
+	zeroTime = time.Time{}
+
+	log = logging.LoggerForModule()
+)
+
+func writeCSV(metrics []*storage.Usage, iow io.Writer) error {
+	csvWriter := csv.NewWriter(iow)
+	csvWriter.UseCRLF = true
+
+	record := []string{"Timestamp", "Nodes", "CPU Units"}
+	if err := csvWriter.Write(record); err != nil {
+		return errors.Wrap(err, "failed to write CSV header")
+	}
+	for _, m := range metrics {
+		record[0] = protoconv.ConvertTimestampToTimeOrDefault(m.Timestamp, zeroTime).UTC().Format(time.RFC3339)
```

Comment:
nit: I think we should extract `time.RFC3339` to pkg and name it so it will be obvious why it's like this. In other places we have 850, 822, 3339Nano 1123